### PR TITLE
Set cbc to default solver

### PR DESCRIPTION
**Pull request by Crissal1995**
I've come across a bug; on Linux (but I think it's multi os) I installed only mip and, when launching my script using it, I got a `NameError: name 'ffi' is not defined. `
I searched where the problem was (line 264 of gurobi.py file, called in line 485 of model.py` __init__`), tried to fix it, but the problem was that in the first try except block ffi was initializated AFTER the raise of exception (going into the except block), causing the has_gurobi to be False and ffi to not exist in the workspace.
TL;DR setting the solver_name to cbc (the one that comes included with installation btw) there isn't any problem and everything works

### CODE REVIEW

**Comment by h-g-s** [comment]
Hi, are you using the last Python-MIP version (1.3) ? I think that in 
previous versions cffi was not included as a dependency, not it is and I 
think that installing it will also install cffi.  Could you check it ?

-- 
=============================================================
Haroldo Gambini Santos
Computing Department
Universidade Federal de Ouro Preto - UFOP
email: haroldo@ufop.edu.br
        Haroldo.GambiniSantos@cs.kuleuven.be
home/research page: www.decom.ufop.br/haroldo


It has long been an axiom of mine that the little things are infinitely
the most important.
-- Sir Arthur Conan Doyle, "A Case of Identity"

**Comment by tkralphs** [comment]
If you look at the fork, the parent commit to the PR is d88a1da93f7af4de3739d3457d2bbd274cb6ced2, which is 1.3.

**Comment by h-g-s** [comment]
It was really a bug for detecting gurobi. I applied a version of your 
patch (without the formatting changes) and released 1.3.1.   Thanks a lot 
for the fix.

-- 
=============================================================
Haroldo Gambini Santos
Computing Department
Universidade Federal de Ouro Preto - UFOP
email: haroldo@ufop.edu.br
        Haroldo.GambiniSantos@cs.kuleuven.be
home/research page: www.decom.ufop.br/haroldo


It has long been an axiom of mine that the little things are infinitely
the most important.
-- Sir Arthur Conan Doyle, "A Case of Identity"

**Comment by Crissal1995** [comment]
You're welcome!
